docs/supported_platforms.md

```diff
@@ -22,6 +22,9 @@ TODO document this platform
 
 TODO document this platform
 
+* `base_domain` in `ENV_DATA` section have to be properly configured for
+  particular deployment (see for example `conf/ocsci/azure_qe_rh_ocs_com_domain.yaml` file)
```

Comment:
I suggest to be more clear what this actually means, and note this instead:

To deploy ocs-ci cluster on Azure platform using one of OCS QE subscriptions, you need to specify one deployment config file from `conf/deployment/azure/` directory (which one to use depends the the type of deployment one needs to use) and also one config file with azure `base_domain` value (either `onf/ocsci/azure_qe_rh_ocs_com_domain.yaml` or `onf/ocsci/azure2_qe_rh_ocs_com_domain.yaml` file), because each azure subscription has it's own domain name.

Reply:
Thanks, I'll incorporate this.